admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -43,7 +45,7 @@ export const ApplicationPage: React.FunctionComponent = () => {
             return applicationTypesQuery.payload.value.bundleId;
         }
 
-        return undefined;
+        return 'bundle';
```

Comment:
Why is this needed? If is because the link, perhaps you could change it to an empty string in the link.

Reply:
It was because of the link. Will change to your suggestion below.